topics/introduction/tutorials/galaxy-intro-101-everyone/tutorial.md

```diff
@@ -0,0 +1,667 @@
+>  > >
+>  > > 3. Your new output dataset will look something like this:
+>  > >
+>  > >   ![Contents of the `Group` output dataset](../../images/101_foreveryone_scatter_petal.png)
+>  > >
+>  > > We can better differentiate between the 3 Iris species but for some samples the petal length versus width is still insufficient
+>  > > to differentiate *Iris versicolor* from *Iris virginica*. And as before, *Iris setosa* can easily be distinguished from the two other species.
+>  > {: .solution}
+>  {: .question}
+{: .hands_on}
+
+# Galaxy management
+
+## Convert your analysis history into a workflow
+
+When you look carefully at your history, you can see that it contains all steps of our analysis, from the beginning to the end. By building this history we have actually built a complete record of our analysis with Galaxy preserving all parameter settings applied at every step. But when you receive new data, or a new report is requested, it would be tedious to do each step over again. Wouldn't it be nice to just convert this history into a workflow that we will be able to execute again and again?
+
+Galaxy makes this very easy with the `Extract workflow` option. This means any time you want to build a workflow, you can just perform the steps once manually, and then convert it to a workflow, so that next time it will be a lot less work to do the same analysis.
+
+> ### {% icon hands_on %} Hands-on: Extract workflow
+>
+> 1. **Clean up** your history.
+>    - If you had any failed jobs (red), please remove those datasets from your history by clicking on the `x` button.
+>    - This will make the creation of a workflow easier.
+>
+> 2. Go to the **History Options menu**  {% icon galaxy-gear %} menu
+>    - Select the **Extract Workflow** option.
+>
+>      ![`Extract Workflow` entry in the history options menu](../../images/history_menu_extract_workflow.png)
+>
+>    - The central panel will change as shown below and you will be able to choose which steps to include/exclude and how to name the newly created workflow.
+>
+>    ![Selection of steps for `Extract Workflow` from history](../../images/101_foreveryone_workflow_create.png)
+>
+> 3. **Rename** the workflow to something descriptive
+>    - For example: `Exploring Iris dataset with statistics and scatterplots`.
+>    - If there are any steps that shouldn't be included in the workflow, you can **uncheck** them.
+>
+> 4. Click on the **Create Workflow** button near the top.
+>    - You will get a message that the workflow was created. But where did it go?
+>
+> 5. Click on **Workflow** in the top menu of Galaxy.
+>    - Here you have a list of all your workflows.
+>    - Your newly created workflow should be listed at the top:
+>
+>    ![`Where workflows go` list](../../images/101_foreveryone_workflow.png)
+>    ![`Your workflows` list](../../images/101_foreveryone_workflow_list.png)
+{: .hands_on}
+
+## The workflow editor
```

Comment:
Thanks a lot @nagoue for this workflow section! 2 ideas:
- You do a lot of renaming of datasets in this tutorial (which is a good habit to teach), shall we do the same in the workflow, add output renaming for the datasets we don't hide?
- Do I understand correctly that different columns should be used for the new dataset? should we make these "set at runtime" parameters?

this could be 2 nice features to show, what do you think?

Reply:
@shiltemann I tried to address your two points but I was not sure about `renaming of datasets`  in the workflow. Let me know if what I have done is not correct! Thanks for your feedback.

Reply:
Hi @annefou thanks a lot! sure, no worries, I will have a look at the renaming step :)